### DISCUSSION TITLE
Sprites needed

### DISCUSSION THREAD

**Discussion by TobleroneSwordfish** [stonepillars/goon-flock#26] [General]
Flock ~~wall, window~~ and door sprites need updating to perspective wall style, file is `icons/misc/featherzone.dmi`

**Comment by amylizzle** [comment]
~~Flock converted computers~~ - DONE
Flock converted mainframes

Until someone does a proper job, I suggest we just grab the existing sprites for these and make them teal. Maybe sparkly.

**Reply by TobleroneSwordfish** [reply]
Makes sense to make all computers convert to the same flock-thing, that way it's just one sprite for computers and one for the mainframe.

**Reply by amylizzle** [reply]
They ought to do different things though - the mainframe is rarer and harder to capture, and should have a unique sprite IMO

**Comment by amylizzle** [comment]
Gnesis turret:
I'm basically imagining a covered glass tank covered in crystalline needles, with coagulated gnesis bubbling inside.
The in-game description currently reads  "A vat of bubbling teal fluid, covered in hollow spikes."

It'll need a couple states: offline, refilling, and idle. It'd also be great if the sprite showed the internal fluid level, but that might be hard.

Behaviour is that when powered, it generates gnesis internally up to a max of 250 units. If there's a target in range and there's enough gnesis in the tank, it fires a spike of nanites (see: syringe dart) into the target with a puff of air.

**Comment by TobleroneSwordfish** [comment]
~~Flock floor light, right now they all convert to wall sprites.
Relevant file: icons/obj/lighting.dmi~~

Done!

**Reply by amylizzle** [reply]
Floor light done (but feel free to improve it if you want)

**Comment by FlameArrow57** [comment]
Flockbits - dormant state, damaged states

**Comment by FlameArrow57** [comment]
Atom icon for mark for deconstruction

**Comment by FlameArrow57** [comment]
Turf conversion icons - fibrenets, barricades

**Comment by FlameArrow57** [comment]
Waiting task icon for tgui panel

**Reply by TobleroneSwordfish** [reply]
TGUI icons aren't sprites, they come from https://fontawesome.com/v5/search?m=free
Though I would love if we had sprites for every AI task

**Reply by amylizzle** [reply]
maybe check the licensing, but we could potentially save them as icons for use in other things

**Comment by FlameArrow57** [comment]
Abilities - Unique ability icon for partition mind, unique icon for flock control panel (or radio stun burst, but icon seems fine for radio stun burst), rotated icon for designate priority tile (looks like it's wrong orientation?)

**Comment by FlameArrow57** [comment]
Rift could maybe use a resprite? Doesn't seem very rift like, but idk

**Comment by FlameArrow57** [comment]
New perspective walls need animated and broken states

**Comment by FlameArrow57** [comment]
Fun stuff - flock flora/plants (if in theme of flock?)
